plugins/modules/ini_file.py

```diff
@@ -179,6 +197,28 @@
       - pepsi
     mode: '0600'
     state: present
+
+- name: remove the peer configuration for 10.128.0.11/32
```

Comment:
is the CIDR in this comment correct? I would have thought it ought to be `10.4.0.11/32`

Reply:
corrected